getting-started/mbed_profile.json

```diff
@@ -0,0 +1,16 @@
+{
+    "ARMC6": {
+        "common": ["-c", "--target=arm-arm-none-eabi", "-mthumb", "-O1",
+                   "-Wno-armcc-pragma-push-pop", "-Wno-armcc-pragma-anon-unions",
+                   "-Wno-reserved-user-defined-literal", "-Wno-deprecated-register",
+                   "-DMULADDC_CANNOT_USE_R7", "-fdata-sections",
+                   "-fno-exceptions", "-MMD", "-D_LIBCPP_EXTERN_TEMPLATE(...)=",
+                   "-fshort-enums", "-fshort-wchar", "-DMBED_TRAP_ERRORS_ENABLED=1"]
+    },
+    "IAR": {
+        "common": [
+            "--no_wrap_diagnostics", "-e",
+            "--diag_suppress=Pa050,Pa084,Pa093,Pa082,Pe540", "-Ol", "--enable_restrict",
+             "-DMBED_TRAP_ERRORS_ENABLED=1"]
```

Comment:
When using this profile, I get errors due to variable length arrays not being enabled. I'd suggest adding the rest of the mbed-os profile information:
```
        "asm": [],
        "c": ["--vla", "--diag_suppress=Pe546"],
        "cxx": ["--guard_calls", "--no_static_destruction"],
        "ld": ["--skip_dynamic_initialization", "--threaded_lib"]
```